### ISSUE TITLE
DeprecationWarning on Django 3.0+

### ISSUE THREAD

**Issue by adamchainz** [laymonage/django-jsonfield-backport#33]
I found this on Django 3.0 when integrating in Django-MySQL:

```
File ".../django_jsonfield_backport/apps.py", line 10, in JSONFieldConfig
  verbose_name = _("JSONField backport from Django 3.1")
File ".../django/utils/translation/__init__.py", line 142, in ugettext_lazy
  warnings.warn(
django.utils.deprecation.RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
```

I run my library tests with `python -W error::DeprecationWarning`, to avoid passing warning errors on to users, so this broke my test run.

**Comment by laymonage** [comment]
Fix released in 1.0.4 :tada:

**Comment by adamchainz** [comment]
Thank you!
